controllers/home.go

```diff
@@ -14,8 +16,16 @@ func (h *HomeController) applyRoutes(engine *gin.Engine) {
 }
 
 func (h *HomeController) displayHome(c *gin.Context) {
-	c.HTML(http.StatusOK, "home", gin.H{
-		"title":   "Home",
-		"flashes": session.Flashes(c),
+	userID := session.Get(c, "user_id")
+
+	var user models.User
+	if userID != nil {
+		user, _ = models.FindOneUserByID(userID)
+	}
```

Comment:
Since pretty much all authenticated controller would need the user data, what do you think to later (in another task) move this logic into a middleware e.g. `current_user.go`?

Reply:
Indeed, I will move this into a middleware in another task